src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -997,25 +997,30 @@ module MutRecBindingChecking =
                     for b1, b2 in List.pairwise defnAs do
                         match b1, b2 with
                         | TyconBindingPhase2A.Phase2AMember {
-                            SyntacticBinding = NormalizedBinding(valSynData = SynValData(transformedFromProperty = Some getPropertyIdent; memberFlags = Some mf))
+                            SyntacticBinding = NormalizedBinding(pat = SynPat.Named(ident = SynIdent(ident = getIdent)); valSynData = SynValData(memberFlags = Some mf))
```

Comment:
`transformedFromProperty` was removed from `SynValData`.
We can draw the same conclusion that the members were originally a property when the range of the `getIdent` and `setIdent` are the same.